docs/changelog/1444.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+Separate the environment variable that let's tox know it is invoked in
+the parallel mode (``_TOX_PARALLEL_ENV``) from the variable that informs
+the tests that tox is running in parallel mode (``TOX_PARALLEL_ENV``).
+— by :user:`hroncok`
```

Comment:
I feel like you should mention here what this delivers to users; aka allowing to run nested parallel executions?